packages/completer/src/handler.ts

```diff
@@ -554,6 +556,9 @@ export namespace CompletionHandler {
      * The only method of this connector that will ever be called is `fetch`, so
      * it is acceptable for the other methods to be simple functions that return
      * rejected promises.
+     *
+     * @deprecated will be removed, or will return `CompletionHandler.ICompletionItemsConnector`
+     * instead of `IDataConnector` in future versions
```

Comment:
This description seems to have been copied from a getter, but this is an option value which gets set, so it needs to be adjusted. Something like:

```suggestion
     * @deprecated passing `IDataConnector` is deprecated;
     * pass `CompletionHandler.ICompletionItemsConnector`
```

(may require a lint pass).